packages/commonwealth/client/scripts/views/pages/CreateCommunity/components/ActionSteps/ActionStep.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { CWIcon } from 'views/components/component_kit/cw_icons/cw_icon';
+import { CWText } from 'views/components/component_kit/cw_text';
+import CWCircleSpinner from 'views/components/component_kit/new_designs/CWCircleSpinner';
+import { CWButton } from 'views/components/component_kit/new_designs/cw_button';
+
+import './ActionStep.scss';
+
+export interface ActionStepProps {
+  state: 'not-started' | 'completed' | 'loading';
+  label: string;
+  index: number;
+  actionButton?: {
+    label: string;
+    disabled: boolean;
+    onClick: () => void;
+  };
+}
```

Comment:
maybe we could move this to a `types.ts` file in the same folder?

Reply:
moved